akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowZipWithIndexSpec.scala

```diff
@@ -20,8 +24,10 @@ class FlowZipWithIndexSpec extends StreamSpec {
 
     "work in the happy case" in assertAllStagesStopped {
       val probe = TestSubscriber.manualProbe[(Int, Long)]()
-      Source(7 to 10).zipWithIndex.runWith(Sink.fromSubscriber(probe))
-
+      //#zip-with-index
+      Source(7 to 10).zipWithIndex
+      //#zip-with-index
+        .runWith(Sink.fromSubscriber(probe))
```

Comment:
Would be good with an aligned example, so maybe just duplicate the Java one in Scala after improving it.